configs/det/dbnet/db_r18_td500.yaml

```diff
@@ -135,8 +135,11 @@ eval:
           img_mode: RGB
           to_float32: False
       - DetLabelEncode:
-      - ScalePadImage:
+      - DetResize:  #ScalePadImage
           target_size: [ 800, 800 ] # h, w
+          padding: True
+          limit_type: None
+          force_divisable: False
```

Comment:
add explicit `keep_ratio: True` to emphasize that it's scaling rather than resizing. Do the same for other replaced `ScalePadImage`.